explorer/explorertypes.go

```diff
@@ -67,9 +67,16 @@ type TxInfo struct {
 }
 
 type TicketInfo struct {
-	TicketMaturity int64
-	PoolStatus     string
-	SpendStatus    string
+	TicketMaturity               int64
+	TimeTillMaturity             float64 //TimeTillMaturity represents the time before a particular ticket reaches maturity
+	PoolStatus                   string
+	SpendStatus                  string
+	TicketPoolSize               int64   //TicketPoolSize represents the total number of block before the average win
```

Comment:
Not what TicketPoolSize necessarily.    When taken from chaincfg.Params, it's actually 8192 (40960 / 5), where the "ticket pool" size in tickets is 40960.  The meaning of 8192 is the "number of different ticket numbers".  The idea being that in the ticket pool, 5 tickets at a time are drawn, so you can conceptualize the pool of 40960 tickets as being in 8192 buckets.

If that's not clear, we should consider making `TicketPoolSize` equal to `exp.ChainParams.TicketPoolSize * TicketsPerBlock`, and then compute average time to vote as `TicketPoolSize / TicketsPerBlock`.  It's extra math, but the value of `TicketsPerBlock` is perhaps more intuitive that way.

Reply:
I'm confused here wouldn't this evaluate back to exp.ChainParams.TicketPoolSize, and what do you mean by average time to vote is it the same as average win time

Reply:
Because average win time is the how close the ticket is to voting in days

Reply:
Yes, it would evaluate back to exp.ChainParams.TicketPoolSize.  But what I'm pointing out is just that `TicketPoolSize` in the `TicketInfo` struct is either named wrong (should be avg blocks to vote) or the value should be set to exp.ChainParams.TicketPoolSize * exp.ChainParams.TicketsPerBlock.  The units of exp.ChainParams.TicketPoolSize are not in number of tickets, so it's a really misleading variable.

On mainnet, the target ticket pool size is 40960 tickets.  We can estimate number of votes as 5/block, not counting misses.  So it works out that the average time to vote is 8192 = 40960/5 blocks.

Reply:
But a related issue I brought up with @raedah is that there's a different point where your ticket has a 50% chance to vote by. which is not the same as this.  It's a Poisson distribution, not a Gaussian.  So over all tickets, the average time to vote may be 8192, but a given ticket has a 50% change of voting by a time closer to block 5700.

Reply:
I don't seem to understand how to calculate it, Could you please explain the formula